bin/docker_start

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+require 'uri'
+require 'active_record'
+
+# check that postgres is up
+pg_uri = URI.parse(ENV['DATABASE_URL'])
+port = pg_uri.port || 5432
+
+unless system("dockerize -wait tcp://#{pg_uri.host}:#{port} -timeout 60s")
```

Comment:
Do we have to wait for the database to start and the database user to exist if we're not running in `AUTO_MIGRATE` mode? I think we're better off not doing any of this waiting in production (especially since the database can disappear at any point while the app is up and the process won't crash).